core/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -332,7 +332,7 @@ class IntegrationSpec extends SpecBase(kafkaPort = KafkaPorts.IntegrationSpec) {
       }
     }
 
-    "call the onRevoked hook" in {
+    "call the onRevoked hook" ignore {
```

Comment:
Why is this ignored?

Reply:
This test was ignored because of the new policy about re-use partitions. I've re-activated the test again. Now, we have to wait for the default time to ensure that the old partition is closed and a new one will be created.